Gemfile.lock

```diff
@@ -100,4 +100,4 @@ DEPENDENCIES
   simplecov-cobertura
 
 BUNDLED WITH
-   2.0.1
```

Comment:
Shame the Ruby version doesn't include [bundler autoswitch](https://bundler.io/guides/bundler_2_upgrade.html#version-autoswitch) to help automatically switching between desired versions.

Reply:
Yeh :(